### PR TITLE
added House Class, ready to merge with master

### PR DESCRIPTION
**Pull request by Lion1Blue**

### CODE REVIEW

File: PullRequest_Demo/House.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PullRequest_Demo
+{
+    class House
+    {
+        List<Person> residents = new List<Person>();
```
Comment (Lion1Blue) [inline PullRequest_Demo/House.cs:11]: this should be an Array

**Review by Lion1Blue — COMMENTED** [review]
this should be an Array

**Review by Lion1Blue — COMMENTED** [review]
you should change the List Datastructure to an Array

**Review by Lion1Blue — COMMENTED** [review]
everything is fine

File: PullRequest_Demo/House.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+        public House(List<Person> residents)
+        {
+            this.residents = residents;
+        }
+
+        public void AddResidents(Person person)
+        {
```
Comment (Fazil-edu) [inline PullRequest_Demo/House.cs:19]: no residents should be added

**Comment by Fazil-edu** [comment]
the residents are not accepted

**Pull request closed without merging**